docs/disasterRecoverInstructions.md

```diff
@@ -0,0 +1,79 @@
+### Disaster Recovery Instructions
+
+
+
+## Deploy Devhub on OpenShift
+Learning from build.js in pipeline/lib file about its’ deploying process, here’s my approach of manually deploy Devhub. 
```

Comment:
These instructions are to get the Devhub stood up in a prompt fashion. This will get the application running but it will not standup supporting infrastructure such as jenkins. It is __HIGHLY__ recommended you follow instructions for the pipeline-cli/bcdk to stand up a pipeline to manage the deployment/builds of Devhub. 

As another tip, use the `--dry-run` flag with any `oc apply` command to ensure that your cli commands are working as expected.